lib/Notification/styles.scss

```diff
@@ -1,9 +1,7 @@
 .notification {
-	font-size: 13px;
-	line-height: 1.75em;
-	padding: 10px 15px;
+	font-size: $typo-size-slight;
+	padding: $layout-spacing-base/2 $layout-spacing-base*0.75;
```

Comment:
Going to hold the merging of this PR then until we discuss that strategy so we can come back to this :)

I know what you mean about only using the base variables to perform the calculations, and I'm all up for that, but these edge cases really complicate things (like finding factors of multiplication to achieve the same effect. So far I've found calculations like `*2` or `/2` to be pretty maintainable, but multiplying by different factors can sometimes be hard to justify (what if a component calls for `* 0.65` in order to look alright?)

A chat for us to have later on!